### PR TITLE
Fix memcached service type

### PR DESCRIPTION
**Pull request by goodspark**
The type `memcached` shows up incorrectly in APM. So either the trace clients need to send what APM expects or the APM product needs to map that to `cache`. Either are fine by me.

screenshot from 2018-08-29 21-07-36

screenshot from 2018-08-29 21-23-49

### CODE REVIEW

File: ddtrace/ext/memcached.py [hunk 1]
```diff
@@ -1,4 +1,4 @@
 CMD = "memcached.command"
 SERVICE = "memcached"
-TYPE = "memcached"
+TYPE = "cache"
```
Comment (goodspark) [inline ddtrace/ext/memcached.py]: Interesting. This used to be `AppTypes.cache` but was changed in the same PR that added memcached support. https://github.com/DataDog/dd-trace-py/pull/511#discussion_r207932657

**Comment by Kyle-Verhoog** [comment]
@goodspark this should be fixed in the product now. Can you confirm on your end?

**Comment by goodspark** [comment]
@Kyle-Verhoog Right now I see memcached labeled as "db". Shouldn't it be "cache"?

**Comment by goodspark** [comment]
Ah, I see! Thank you for the follow up!

**Comment by palazzem** [comment]
Hello again @goodspark ! We're going to close this PR because the problem was notified internally and we're working to make `memcached` a supported Cache type in our frontend. I'll reach you again once the problem is entirely solved.

Thank you very much!

**Comment by goodspark** [comment]
Thank you!

**Pull request closed without merging**